### PR TITLE
Slash is a Slack terminal client written in Swift.

### PR DESCRIPTION
**Pull request by konradrodzik**
Initial release of slash 0.1.0

Basic chat functionality over Real Time Messaging API.
Support for Slack message formatting (escaped characters, emoji, mentions).
Support for OAuth2 and API tokens (https://api.slack.com/docs/oauth-test-tokens) authorization methods.
Switch between channels using 'Tab' key.
You can now scroll through messages using up/down arrow keys.

### CODE REVIEW

File: Formula/slash.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Slash < Formula
+  desc "Slack terminal client written in Swift."
+  homepage "https://github.com/slash-hq/slash"
+  url "https://github.com/slash-hq/slash/archive/0.1.0.tar.gz"
+  sha256 "8a709579ffba7c47b1e1975bb418d72ecbd542539d4cd6f7a72d876808bfbdb2"
+
+  depends_on :xcode
```
Comment (MikeMcQuaid) [inline Formula/slash.rb]: `depends_on :xcode => :build` and add a line below this.

File: Formula/slash.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  def install
+    xcodebuild "-workspace", "slash.xcodeproj/project.xcworkspace", "-derivedDataPath", "prefix.to_s", "-configuration", "Release", "-scheme", "slash", "SYMROOT=#{prefix}/Build"
+    bin.install(prefix + "Build/Release/slash")
+  end
+
+  test do
+    system "false"
```
Comment (MikeMcQuaid) [inline Formula/slash.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Formula/slash.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+class Slash < Formula
+  desc "Slack terminal client written in Swift."
+  homepage "https://github.com/slash-hq/slash"
+  url "https://github.com/slash-hq/slash/archive/0.1.0.tar.gz"
+  sha256 "8a709579ffba7c47b1e1975bb418d72ecbd542539d4cd6f7a72d876808bfbdb2"
+
+  depends_on :xcode
+  def install
+    xcodebuild "-workspace", "slash.xcodeproj/project.xcworkspace", "-derivedDataPath", "prefix.to_s", "-configuration", "Release", "-scheme", "slash", "SYMROOT=#{prefix}/Build"
```
Comment (MikeMcQuaid) [inline Formula/slash.rb]: Split across multiple lines for readability.

Comment (MikeMcQuaid) [inline Formula/slash.rb]: `"prefix.to_s"` should be just `prefix`

File: Formula/slash.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    bin.install(prefix + "Build/Release/slash")
```
Comment (MikeMcQuaid) [inline Formula/slash.rb]: `bin.install "Build/Release/slash"`

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]

**Comment by MikeMcQuaid** [comment]
This project is a day old. Please fill out the PR template in future which details the criteria (i.e. in this case `brew audit --new-formula slash` would point out many issues). Passing on this until it's more popular. Please consider [creating your own tap](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md#how-to-create-and-maintain-a-tap) instead. Thanks!

**Comment by ilovezfs** [comment]
If anyone is interested in trying it, it looks like it's been checked in here: https://github.com/slash-hq/slash/commit/3bf772dc806a5b8a2e94e7726da25dce56b68dc2

**Pull request closed without merging**